README.md

```diff
@@ -84,22 +97,104 @@ python3 run.py --test riscv_page_table_exception_test --co
 ....
 ```
 
+### CSR Generation
+
+
+The CSR generation script is located at [scripts/gen_csr_test.py](https://github.com/google/riscv-dv/blob/master/scripts/gen_csr_test.py).  To quickly generate a basic CSR test, run the below command:
```

Comment:
Please add some description about the intent and procedure of the privileged CSR test.